package.json

```
@@ -59,11 +60,13 @@
     "@adobe/helix-deploy-plugin-webpack": "^1.0.2"
   },
   "dependencies": {
-    "@adobe/fastly-native-promises": "3.0.19",
+    "@adobe/fastly-native-promises": "3.1.0",
     "@fastly/js-compute": "3.35.2",
     "chalk-template": "1.1.2",
     "constants-browserify": "1.0.0",
+    "esbuild": "^0.25.0",
```

```suggestion
    "esbuild": "0.25.0",
```

prefer exact versions